twitter/visualizations/render_graphs.py

```diff
@@ -3,7 +3,9 @@
 
 import twitter.visualizations.graphs as graphs
 
-from twitter.processing.tweet_processing import get_tweets_from_json_file, create_pd_from_tweets, filter_covid_tweets
+from twitter.processing.tweet_processing import create_pd_from_tweets, filter_covid_tweets, create_tweet_for_df
+from twitter.data.fetch_from_api import fetch_tweets
+
 
 TITLE_TO_MODE = {
```

Comment:
I think, later on, we shall have to put this TITLE_TO_MODE dict in a central place because I also need to use it. For now, I've been using a hacky workaround.
PS: This is not to be updated immediately in this particular PR: it's a suggestion as we move forward.

Reply:
Ok, we can always put it in the `constants.py` file

Reply:
Cool